packages/opentelemetry-exporter-prometheus/src/prometheus.ts

```diff
@@ -0,0 +1,280 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ExportResult } from '@opentelemetry/base';
+import { NoopLogger } from '@opentelemetry/core';
+import {
+  LabelValue,
+  MetricDescriptor,
+  MetricDescriptorType,
+  MetricExporter,
+  ReadableMetric,
+} from '@opentelemetry/metrics';
+import * as types from '@opentelemetry/types';
+import { createServer, IncomingMessage, Server, ServerResponse } from 'http';
+import * as Prometheus from 'prom-client';
+import * as url from 'url';
+import { ExporterConfig } from './export/types';
+
+export class PrometheusExporter implements MetricExporter {
+  static readonly DEFAULT_OPTIONS = {
+    port: 9464,
+    startServer: false,
```

Comment:
Is there any way to start the server later if I use this default value at start?

Reply:
You can call public `startServer` method